stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -50,6 +51,11 @@ public class MessageInputView : ConstraintLayout {
 
     private var onSendButtonClickListener: OnMessageSendButtonClickListener? = null
     private var typingListener: TypingListener? = null
+
+    public val attachmentBiggerThanMax: StateFlow<Boolean> by lazy {
```

Comment:
Do not like to expose StateFlow in a view. We can pass just some simple handler like:
``` kotlin
fun interface BigFileSelectionHandler {
   fun handleBigFileSelected(fileSize: Int) 
}
```

It let us not to expose any state and don't use `StateFlow` that could be not so convenient for Java users.